tsdb/wlog/wlog.go

```diff
@@ -38,8 +38,8 @@ import (
 )
 
 const (
-	DefaultSegmentSize = 128 * 1024 * 1024 // 128 MB
-	pageSize           = 32 * 1024         // 32KB
+	DefaultSegmentSize = 128 * 1024 * 1024 // DefaultSegmentSize is 128 MB.
+	pageSize           = 32 * 1024         // pageSize is 32KB.
```

Comment:
This is worse.  Can we turn off the check that made you do it?

Reply:
I don't feel strongly whether this is worse or better.

However, if we turn off the check that made me do this, we won't check at all that doc comments start with the name they are commenting.

If it is really important to keep things here as they were, I would propose to add a nolint pragma.